pkg/cmd/codespace/create.go

```diff
@@ -89,6 +91,37 @@ func (a *App) Create(ctx context.Context, opts createOptions) error {
 		branch = repository.DefaultBranch
 	}
 
+	devContainerPath := opts.devContainerPath
+
+	// now that we have repo+branch, we can list available devcontainer.json files (if any)
+	if len(opts.devContainerPath) < 1 {
```

Comment:
I think a more idiomatic way to check this would be 
```suggestion
	if opts.devContainerPath == "" {
```